auth-api/src/auth_api/services/affiliation_invitation.py

```diff
@@ -767,9 +767,7 @@ def check_auth_for_invitation(
         invitation_type = invitation_type or AffiliationInvitationType.from_value(invitation.type)
         from_org_id = from_org_id or invitation.from_org_id
         match invitation_type:
-            case AffiliationInvitationType.REQUEST:
-                check_auth(org_id=from_org_id, one_of_roles=(ADMIN, COORDINATOR, STAFF))
-            case AffiliationInvitationType.EMAIL:
+            case AffiliationInvitationType.REQUEST | AffiliationInvitationType.EMAIL:
```

Comment:
This enables:
CREATE - Create an affiliation invitation
UPDATE - Expire only
DELETE - Soft delete